### PR TITLE
Update Plugwise instruction

### PR DESCRIPTION
**Pull request by bouwew**
## Proposed change

Update describing the added functionality via Core PR #103360

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/103360
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/plugwise.markdown [hunk 1]
```diff
@@ -82,17 +82,26 @@ script:
...
+#### Set HVAC mode
 
 Service: `climate.set_hvac_mode`
 
-Available options include `auto`, `heat`, and `cool` (only when there is a cooling option available). The meaning of `auto` is that a schedule is active and the thermostat will change presets accordingly. The meaning of `heat/cool` is that there is no schedule active, i.e., the active preset or manually set temperature is used to control the climate in your house or rooms.
+Available options include `off` (Adam only) `auto`, `cool`, `heat`, and `heat_cool` (Anna with Elga only).
+
+The meaning of `off` is that the Adam regulation is set to off. This means that the connected HVAC-system does not heat or cool, only the domestic hot water heating function, when available, is active.
+
+The meaning of `cool` or `heat` is that there is no schedule active, i.e. the system is manually set to cooling- or heating-mode, the system will be active if the room temperature is above/below the thermostat setpoint.
```
Comment (c0ffeeca7) [inline source/_integrations/plugwise.markdown]:
```suggestion
The meaning of `cool` or `heat` is that there is no schedule active. For example, if the system is manually set to cooling- or heating-mode, the system will be active if the room temperature is above/below the thermostat setpoint.
```

File: source/_integrations/plugwise.markdown [hunk 2]
```diff
@@ -82,17 +82,26 @@ script:
...
+
+The meaning of `heat/cool` is that there is no schedule active, i.e. the system is in automatic cooling- or heating-mode, the active preset or manually set temperature is used to control the HVAC-system.
```
Comment (c0ffeeca7) [inline source/_integrations/plugwise.markdown]:
```suggestion
The meaning of `heat/cool` is that there is no schedule active. For example, if the system is in automatic cooling- or heating-mode, the active preset or manually set temperature is used to control the HVAC system.
```

Comment (bouwew) [reply source/_integrations/plugwise.markdown]: Thanks for reviewing!

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @bouwew :+1:

**Pull request merged**